SolidStateGroup/bullet-train-api
# feat: versioned change request change sets

Thanks for submitting a PR! Please check the boxes below:


## Changes

Adjusts the way change requests are created in order to prevent versioned change requests from overwriting each other.

## How did you test this code?

- Create a change request with a segment override
- Create a change request with a remote config update
- Publish the segment override CR
- Check the diff of the second CR, it should recognise segment overrides as unchanged
- Fixes an issue where creating / actioning CRs doesn't update the count
